code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -352,6 +352,16 @@
 	siemens_coefficient = 0.9
 	w_class = 3
 	slot = "over"
+	var/allow_tail_hiding = TRUE //in case if you want to allow someone to switch the HIDETAIL var or not
+
+/obj/item/clothing/accessory/poncho/verb/toggle_hide_tail()
```

Comment:
This is not a fix, but a feature which goes into development. Unless @skull132 approves it to slip in.

Reply:
_eh,_ it's mildly related.